### PR TITLE
[Behavioral Analytics] API Docs

### PR DESCRIPTION
**Pull request by joemcelroy**
API Docs for behavioral Analytics
Screenshot 2023-03-23 at 14 25 04
Screenshot 2023-03-23 at 14 24 52
Screenshot 2023-03-23 at 14 24 44
Screenshot 2023-03-23 at 14 24 36
Screenshot 2023-03-23 at 14 24 30

### CODE REVIEW

**Review by afoucret — APPROVED** [review]
Overall good. A small comment on consistency

File: docs/reference/behavioral-analytics/apis/list-analytics-collection.asciidoc [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  },
+  "bar": {
+      "event_data_stream": {
+          "name": "behavioral_analytics-events-bar"
+      }
+  }
+}
+----
+
+The following example returns the a Analytics Collection that matches `foo`:
```
Comment (leemthompo) [inline docs/reference/behavioral-analytics/apis/list-analytics-collection.asciidoc:58]:
```suggestion
The following example returns the Analytics Collection that matches `foo`:
```

File: docs/reference/behavioral-analytics/apis/delete-analytics-collection.asciidoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
+[role="xpack"]
+[[delete-analytics-collection]]
+=== Delete Analytics Collection
+
+++++
+<titleabbrev>Delete Analytics Collection</titleabbrev>
+++++
+
+Removes an Analytics Collection and its associated data-stream.
```
Comment (leemthompo) [inline docs/reference/behavioral-analytics/apis/delete-analytics-collection.asciidoc:9]:
```suggestion
Removes an Analytics Collection and its associated data stream.
```

File: docs/reference/behavioral-analytics/apis/delete-analytics-collection.asciidoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+`400`::
+The `name` was not provided.
+
+`404` (Missing resources)::
+No Analytics Collection matching `name` could be found.
+
+[[delete-analytics-collection-example]]
+==== {api-examples-title}
+
+The following example deletes the Analytics Collection named `my-app`:
```
[the remark this answers is not in the document]
Comment (joemcelroy) [reply docs/reference/behavioral-analytics/apis/delete-analytics-collection.asciidoc:39]: Cool - il update everywhere in delete & put with 'my_analytics_collection' and then for list, will update to underscore

**Review by afoucret — APPROVED** [review]

**Pull request merged**